config/database.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+return [
+    'DB_DATABASE' => __DIR__ . '/../database/database.sqlite',
```

Comment:
@TomasVotruba this is needed due to now require absoute path.

Reply:
It seems only define database type on .env is enough, and will use database/database.sqlite, see:

- https://github.com/rectorphp/getrector-com/pull/1963